src/Optimizely/Optimizely.php

```diff
@@ -588,7 +588,11 @@ public function getFeatureVariableValueForType(
         } else {
             $experiment_id = $decision->getExperimentId();
             $variation_id = $decision->getVariationId();
-            $experiment = $this->_config->getExperimentFromId($experiment_id);
+            if ($decision->getSource() == FeatureDecision::DECISION_SOURCE_ROLLOUT) {
+                $experiment = $this->_config->getRolloutRuleFromId($experiment_id);
+            } else {
```

Comment:
Add else if  ($decision->getSource() == FeatureDecision::DECISION_SOURCE_EXPERIMENT)
In future, if we introduce another source for feature decision. This will save a headache